thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/THttpServiceBuilder.java

```diff
@@ -190,6 +192,17 @@ public THttpServiceBuilder maxRequestContainerLength(int maxRequestContainerLeng
         return this;
     }
 
+    /**
+     * Sets whether the service executes service methods using the blocking executor. By default, service
+     * methods are executed directly on the event loop for implementing fully asynchronous services. If your
+     * service uses blocking logic, you should either execute such logic in a separate thread using something
+     * like {@link Executors#newCachedThreadPool()} or enable this setting.J
```

Comment:
```suggestion
     * like {@link Executors#newCachedThreadPool()} or enable this setting.
```
Awesome comment! remove type 😄